### PR TITLE
App

### PR DESCRIPTION
**Pull request by bradleyzazz**

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import $ from 'jquery';
+import Calendar from './Calendar.jsx'
+
+class App extends React.Component {
+  constructor() {
```
Comment (lheddleston) [inline client/src/components/App.jsx]: Why aren't you passing props into your constructor? Does it behave differently?

Comment (bradleyzazz) [reply client/src/components/App.jsx]: I don't have any props to pass for this one. At least not at this time

File: client/src/components/Calendar.jsx [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    let lessThan = '<';
+    let greaterThan = '>';
+
+    return (
+      <div>
+        <table className="calendar">
+          <thead>
+            <tr>
+              <td colSpan="2">
+                <span onClick={(e) => { this.prevMonth() }}>{lessThan}</span>
```
Comment (lheddleston) [inline client/src/components/Calendar.jsx]: The naming with just the letter 'e' makes it a little hard to read through :)

File: client/src/components/Calendar.jsx [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+  }
+
+  render() {
+    let weekdays = this.weekdaysMin.map(day => {
+      return (
+        <td key={day}>{day}</td>
+      )
+    });
+
+    let emptyDays = [];
```
Comment (lheddleston) [inline client/src/components/Calendar.jsx]: Wow! Your calendar is so organized and readable 👍

File: client/src/components/App.jsx (hunk 1 above)
Comment (lheddleston) [reply client/src/components/App.jsx]: 👍

**Pull request merged**